service.go

```diff
@@ -37,8 +37,8 @@ var servicePrefix = &irc.Prefix{
 
 type serviceContext struct {
 	context.Context
-	nick    string // optional
-	network *network
+	nick    string   // optional
+	network *network // optional
```

Comment:
Nit: this belongs to the previous commit. I can take care of the rebasing if desirable.